frigate/http.py

```diff
@@ -1147,17 +1164,16 @@ def vod_hour_no_timezone(year_month, day, hour, camera_name):
 # TODO make this nicer when vod module is removed
 @bp.route("/vod/<year_month>/<day>/<hour>/<camera_name>/<tz_name>")
 def vod_hour(year_month, day, hour, camera_name, tz_name):
-    tz_offset = int(
+    tz_offset_minutes = int(
         datetime.now(pytz.timezone(tz_name.replace(",", "/")))
         .utcoffset()
         .total_seconds()
         / 60
-        / 60
     )
     parts = year_month.split("-")
     start_date = datetime(
         int(parts[0]), int(parts[1]), int(day), int(hour), tzinfo=timezone.utc
-    ) - timedelta(hours=tz_offset)
+    ) - timedelta(minutes=tz_offset_minutes)
```

Comment:
```suggestion
    ) - datetime.now(pytz.timezone(tz_name.replace(",", "/"))).utcoffset()
```

Reply:
I just realized that utcoffset() is actually a timedelta object already so we don't need to create another one